protos/local/child_workflow.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package coresdk.child_workflow;
+
+import "common.proto";
+import "temporal/api/failure/v1/message.proto";
+
+/**
+ * Used by core to resolve child workflow executions.
+ *
+ * When a child workflow is cancelled, `status` is failed and
+ * failure.cause.info is set to CanceledFailure.
+ */
+message ChildWorkflowResult {
+    oneof status {
+        Success completed = 1;
+        Failure failed = 2;
+    }
+}
+
+/**
+ * Used in ChildWorkflowResult to report successful completion.
+ */
+message Success {
+    common.Payload result = 1;
+}
+
+/**
+ * Used in ChildWorkflowResult to report non successful outcomes such as
+ * application failures, timeouts, terminations, and cancellations.
+ */
+message Failure {
+    temporal.api.failure.v1.Failure failure = 1;
+}
+
+/**
+ * Used by the service to determine the fate of a child workflow
+ * in case its parent is closed.
+ */
+enum ParentClosePolicy {
+    PARENT_CLOSE_POLICY_UNSPECIFIED = 0;
```

Comment:
Should probably say what we default to if left unset -- or will we just not allow unset?

Reply:
It's copied as is from upstream, I'll add a comment.